app/models/mixtape.rb

```diff
@@ -0,0 +1,10 @@
+class Mixtape
+  include ActiveModel::SerializerSupport
+
+  attr_reader :songs
+
+  def initialize(genre:,songs:)
```

Comment:
Could you just use http://api.rubyonrails.org/classes/ActiveModel/Model.html ?

That way can define `genre:,songs:` or just `songs` and an attribute?